### DISCUSSION TITLE
Random system crashes under rosetta

### DISCUSSION THREAD

**Discussion by maddanio** [KhronosGroup/MoltenVK#1307] [General]
This is just to find out wether anyone but me is experiencing random system crashes (i.e. the whole display freezing) when using moltenvk in a intel app on a silicon mac, ie under rosetta 2?

**Comment by baburas** [comment]
Running our application (x86_64) on MacBook Pro with M1 chip also leads to random display freezes. 
Root cause not yet idenfied (also we did not yet invest much time on the problem).
We faced the same issue before on Mac DTK (occurred much more often).

**Reply by maddanio** [reply]
What is mac dtk? I since have had display freeze also without our app, possibly a while after.

**Reply by baburas** [reply]
Developer Transition Kit. The prototype ARM-based Mac computer available for developers before the apple silicon mac's were released.

So far i only had display freeze with our app running (which actually almost always runs)

**Comment by baburas** [comment]
On your side the issue only appears when having an external monitor connected via HDMI.

I found a way to "unfreeze" the Mac: Simply unplug / plugin the HDMI connector -> Display not "frozen" anymore.

**Reply by maddanio** [reply]
For me its main thread
